src/serializer.ts

```diff
@@ -17,14 +19,157 @@ interface PaymentJSON {
  * Provides functionality to serialize from protocol buffers to JSON objects.
  */
 class Serializer {
+/**
+   * Convert a Transaction to a JSON representation.
+   *
+   * @param {proto.Transaction} transaction A Transaction to convert.
+   * @returns {Object} The Transaction as JSON.
+   */
+  public static transactionToJSON(
+    transaction: Transaction
+  ): object | undefined {
+    // Serialize the protocol buffer to a JSON representation.
+    var object: any = transaction.toObject();
+
+    // Convert fields names where direct conversion is possible.
+    this.convertPropertyName("sequence", "Sequence", object);
+    this.convertPropertyName("signingPublicKey", "SigningPubKey", object);
+    this.convertPropertyName(
+      "lastLedgerSequence",
+      "LastLedgerSequence",
+      object
+    );
+
+    // Delete unsupported fields from the protocol buffer.
+    delete object.memosList;
+    delete object.flags;
+    delete object.signature;
+    delete object.signersList;
+    delete object.sourceTag;
+    delete object.accountTransactionId;
```

Comment:
You can do this in a single line:
https://stackoverflow.com/questions/32534602/javascript-built-in-function-to-delete-multiple-keys-in-an-object

Not sure if that's _better_. It might be less clear what's going on, but I tend to like succinct code.

Reply:
Sure. I think that this obfuscates the code meaning a bit, but it's not too complex so I'm fine with tightening up the code a bit.